src/airport.js

```diff
@@ -28,23 +28,22 @@ const airport = {
         {
             return `Airport at full capacity`
         }
-        // if (planesExist())
-        // {
-
-        //     return "This plane is already at the airport"
-        // }
-
         return this.landedPlanes = [...this.landedPlanes, planes]
 
     },
 
     removePlanes: function (planes)
     {
-        if (this.planesExist(planes))
+        return this.landedPlanes.splice(planes, 1);
+    },
+
+    isPlaneAtAirport: function (planes)
```

Comment:
Although the final code this was removed, I should've not added this function from the beginning, it would have saved me time trying to solve all the issues that arose from creating this function. At the end I was able to solve the issues and apply a different approach.